src/class-wc-gateway-btcpay.php

```diff
@@ -294,6 +300,14 @@ public function init_form_fields()
                ),
                 'order_states' => array(
                     'type' => 'order_states'
+               ),
+                'send_country' => array(
+                    'title'       => __('Send country', 'btcpay'),
+                    'type'        => 'checkbox',
+                    'label'       => __('Include customer country in Btcpay invoice</a>', 'btcpay'),
+                    'default'     => 'no',
```

Comment:
Is there any reason this should be `no`?

Reply:
For most btcpay installs the country is not relevant for the btcpay invoice. I'd rather not include information that is not needed.
From the other hand, making the country not-optional lowers the plugin complexity. 

I'm ok with both options, just let me know.

Reply:
Just include it, makes things simpler. Less option is better.